lua/go/output.lua

```diff
@@ -86,16 +94,19 @@ function M.popup_job_result(results, opts)
     local buf_nr = vim.api.nvim_create_buf(false, true)
     vim.api.nvim_buf_set_option(buf_nr, 'bufhidden', 'wipe')
     vim.api.nvim_buf_set_lines(buf_nr, 0, -1, true, results)
+    local actual_content_height = vim.api.nvim_buf_line_count(buf_nr)
     local title = opts.title
-    local top, width = opts.top or 1, opts.width or 80
+    local pos_opts = opts.pos_opts
+	local top = pos_opts.win_height - (math.min(pos_opts.height, actual_content_height) + 1)
```

Comment:
I think we may retrieve `win_height` again with `vim.fn.winheight(0)`. Pass with function return value is weird here.

Reply:
Removed from returned object and fetched again before creating popup.